### PR TITLE
update Ansar Bank

### PR DESCRIPTION
**Pull request by berrely**
Add English name to Ansar Bank (I also added the logo on Wikidata)

### CODE REVIEW

File: data/brands/amenity/bank.json [hunk 1]
```diff
@@ -11510,9 +11510,11 @@
       "tags": {
         "amenity": "bank",
         "brand": "بانک انصار",
+        "brand:en": "Ansar Bank",
```
Comment (LaoshuBaby) [inline data/brands/amenity/bank.json:11513]:
```suggestion
        "brand:en": "Ansar Bank",
        "brand:fa": "بانک انصار",
```

File: data/brands/amenity/bank.json [hunk 2]
```diff
@@ -11510,9 +11510,11 @@
...
         "brand:wikidata": "Q5862675",
         "brand:wikipedia": "fa:بانک انصار",
         "name": "بانک انصار"
```
Comment (LaoshuBaby) [inline data/brands/amenity/bank.json:11516]:
```suggestion
        "name": "بانک انصار",
```

File: data/brands/amenity/bank.json [hunk 3]
```diff
@@ -11510,9 +11510,11 @@
...
+        "name:en": "Ansar Bank"
```
Comment (LaoshuBaby) [inline data/brands/amenity/bank.json:11517]:
```suggestion
        "name:en": "Ansar Bank",
        "name:fa": "بانک انصار"
```

**Review by LaoshuBaby — APPROVED** [review]

**Comment by arch0345** [comment]
Thanks @berrely!

**Pull request merged**